### DISCUSSION TITLE
Hiring Progress for your Location

### DISCUSSION THREAD

**Discussion by AllanRegush** [EddieHubCommunity/support#81] [General]
I'm curious to see how the hiring process is for other locations. Here in Canada we apply directly to the company. I've heard in other countries that people get jobs through recruiters.

**Comment by eddiejaoude** [comment]
In the UK it is a mix of both, we try to avoid recruiters because they are usually a real pain, confuse the situation and blame everyone else

**Comment by RemcoHalman** [comment]
In The Netherlands we have a combination of recruiters and directly applying. I would always recommend applying directly though. Gives of more the personal touch.
